### DISCUSSION TITLE
Add image in background of each cell

### DISCUSSION THREAD

**Discussion by Berachem** [kurkle/chartjs-chart-treemap#162] [General]
Can we add this ? Plz I want to know

**Comment by stockiNail** [comment]
@Berachem the `backgroundColor` is setting the [`fillStyle`](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/fillStyle) property of [CanvasRenderingContext2D](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D). You can pass a [CanvasPattern](https://developer.mozilla.org/en-US/docs/Web/API/CanvasPattern) which is created by an image.

Instead, if you want to add a single image to chart element, you can use a custom plugin to do it. Here is a codepen: https://codepen.io/stockinail/pen/gOKRWdm 
The plugin is adding the image in the center of the element, scaling the image if needed.
